### DISCUSSION TITLE
can pretrained spark params fit into a regular res50 model?

### DISCUSSION THREAD

**Discussion by DietDietDiet** [open-mmlab/mmclassification#1751] [Q&A]
I am using spark as pretrain task, and just wondering if a pretrained spark can be loaded into regular res50 for downstream task.  Thanks @fangyixiao18

**Comment by fangyixiao18** [comment]
Yes, it can be loaded.
You can check this config https://github.com/open-mmlab/mmpretrain/blob/main/configs/spark/benchmarks/resnet50_8xb256-coslr-300e_in1k.py
